src/WiFi.cpp

```diff
@@ -260,11 +260,12 @@ void WiFiClass::handlePingResponse(uint32 u32IPAddr, uint32 u32RTT, uint8 u8Erro
 	}
 }
 
-WiFiClass::WiFiClass()
+WiFiClass::WiFiClass() :
+  _timeout(50000),
```

Comment:
We had a 60000 ms timeout before, I suggest we leave the same value.

Reply:
ok modify and push